### PR TITLE
deque index out of range - fix

### PR DESCRIPTION
**Pull request by lenisko**
Added sleep(0.01) inside __write function.

### CODE REVIEW

File: irc.py [hunk 1]
```diff
@@ -124,6 +124,8 @@ def handle_error(self):
     def __write(self, args, text=None, raw=False):
         # print '%r %r %r' % (self, args, text)
         try:
+            # fix for deque index out of range
+            time.sleep(0.01)
```
Comment (kaneda) [inline irc.py]: I think this is a really bad idea. Thread sleep even would be a bad idea.

**Comment by lenisko** [comment]
It's just 10ms... and... it's working.

**Comment by kaneda** [comment]
@lenisko I don't say this to be discouraging, but this is not an acceptable solution.

**Comment by lenisko** [comment]
Sure, you know better, I'm just novice. But leaving this bug is very bad.

**Comment by myano** [comment]
I'm willing to live with this bug rather than adding a sleep function that will lock up threads and make jenni slower overall. This bug doesn't render the bot useless, rather it just clogs the output of stdout where ever jenni is running.

**Comment by lenisko** [comment]
I needed to do something, after some changes in modules my bot went insane and for about 3/10 commands I got dubles.

Reduced sleep to 1ms, still working fine.

Btw, __write is only IRC client -> server related function?

**Pull request closed without merging**